gpt_json/types_oai.py

```diff
@@ -0,0 +1,60 @@
+from typing import Dict, List, Optional, Union
+
+
+class ChatCompletionDelta:
+    def __init__(
+        self,
+        content: str | None,
+        role: str | None
+    ):
+        self.content = content
+        self.role = role
+
+class ChatCompletionChunkChoice:
```

Comment:
`ChatCompletionDelta`, `ChatCompletionChunkChoice` and `ChatCompletionChunk` have pretty simple implementations here. They should either be `@dataclass` instances or `pydantic.BaseModel` instances.